src/assets/translations/en-US.json

```
@@ -699,6 +703,245 @@
+    "payByLink.linkCreation.fields.country.label": "Country/Region",
+    "payByLink.linkCreation.fields.deliverAt.label": "Delivery date",
+    "payByLink.linkCreation.fields.deliveryAddress.city.label": "City",
+    "payByLink.linkCreation.fields.deliveryAddress.country.label": "Country",
+    "payByLink.linkCreation.fields.deliveryAddress.houseNumberOrName.label": "House number",
+    "payByLink.linkCreation.fields.deliveryAddress.postalCode.label": "Postal code",
+    "payByLink.linkCreation.fields.deliveryAddress.street.label": "Street",
+    "payByLink.linkCreation.fields.description.label": "Description",
+    "payByLink.linkCreation.fields.description.supportText": "Accurate description of customer purchase.",
+    "payByLink.linkCreation.fields.language.label": "Language",
+    "payByLink.linkCreation.fields.linkType.label": "Link type",
+    "payByLink.linkCreation.fields.merchantReference.label": "Merchant Reference",
+    "payByLink.linkCreation.fields.phoneNumber.errors.requiredPhoneCode": "You must select a phone prefix",
+    "payByLink.linkCreation.fields.phoneNumber.errors.requiredPhoneNumber": "You must enter a phone number",
+    "payByLink.linkCreation.fields.sendLinkToShopper.label": "Send the payment request to this email address",
+    "payByLink.linkCreation.fields.sendPaymentSuccessToShopper.label": "Send payment confirmation to this email address.",
+    "payByLink.linkCreation.fields.shopperEmail.error.validEmail": "Please enter a valid email address",
+    "payByLink.linkCreation.fields.shopperEmail.label": "Shopper email",
+    "payByLink.linkCreation.fields.shopperLastName.label": "Shopper last name",
+    "payByLink.linkCreation.fields.shopperName.label": "Shopper name",
+    "payByLink.linkCreation.fields.shopperPhone.label": "Shopper phone",
+    "payByLink.linkCreation.fields.shopperPhone.phonePrefix.placeholder": "Phone prefix",
+    "payByLink.linkCreation.fields.shopperReference.label": "Shopper reference",
+    "payByLink.linkCreation.fields.store.label": "Store",
+    "payByLink.linkCreation.fields.validity.customDuration.error.durationTooLong": "Duration cannot exceed %{maxDays} days",
+    "payByLink.linkCreation.fields.validity.customDuration.error.invalidDurationValue": "Please enter a valid duration value",
+    "payByLink.linkCreation.fields.validity.customDuration.error.missingDurationUnit": "Please select a duration unit",
+    "payByLink.linkCreation.fields.validity.customDuration.error.missingDurationValue": "Please enter a duration value",
+    "payByLink.linkCreation.fields.validity.customDuration.label": "Duration",
+    "payByLink.linkCreation.fields.validity.label": "Validity",
+    "payByLink.linkCreation.fields.validity.linkValidityUnit.custom": "Custom",
+    "payByLink.linkCreation.fields.validity.linkValidityUnit.day": "%{quantity} day",
+    "payByLink.linkCreation.fields.validity.linkValidityUnit.day__plural": "%{quantity} days",
```

WDYT of the following? Like this we don't introduce a new convention.
```suggestion
    "payByLink.linkCreation.fields.validity.linkValidityUnit.days": "%{quantity} days",
```